src/main/java/de/mediathekview/mserver/crawler/hr/tasks/HrSendungsfolgedetailsTask.java

```diff
@@ -2,58 +2,199 @@
+  private static final String THEMA_SELECTOR2 = ".c-programHeader__headline.text__headline";
+  private static final String TITLE_SELECTOR1 = ".c-programHeader__subline.text__topline";
+  private static final String TITLE_SELECTOR2 = ".c-contentHeader__headline";
+  private static final String DATE_TIME_SELECTOR1 = ".c-programHeader__mediaWrapper time";
+  private static final String DATE_TIME_SELECTOR2 = ".c-contentHeader__lead time";
+  private static final String DAUER_SELECTOR1 = ".c-programHeader__mediaWrapper .mediaInfo__byline";
+  private static final String DAUER_SELECTOR2 = ".c-contentHeader .mediaInfo__byline";
+  private static final String DESCRIPTION_SELECTOR1 = ".copytext__text.copytext__text strong";
+  private static final String DESCRIPTION_SELECTOR2 = ".copytext__text";
+  private static final String VIDEO_URL_SELECTOR1 = ".c-programHeader__mediaWrapper source";
+  private static final String VIDEO_URL_SELECTOR2 = "div.videoElement";
+  private static final String UT_URL_SELECTOR1 = ".c-programHeader__mediaWrapper track";
+  private static final String UT_URL_SELECTOR2 = ".c-contentHeader__lead track";
+
+  private static final Type OPTIONAL_ARDVIDEOINFODTO_TYPE_TOKEN = new TypeToken<Optional<ArdVideoInfoDTO>>() {
+  }.getType();
+
+  private final Gson gson;
 
   public HrSendungsfolgedetailsTask(final AbstractCrawler aCrawler,
-      final ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDTOs) {
-    super(aCrawler, aUrlToCrawlDTOs);
+      final ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDtos) {
+    super(aCrawler, aUrlToCrawlDtos);
+
+    gson = new GsonBuilder()
+        .registerTypeAdapter(OPTIONAL_ARDVIDEOINFODTO_TYPE_TOKEN, new HrVideoJsonDeserializer())
+        .create();
+  }
+
+  @Override
+  protected AbstractUrlTask<Film, CrawlerUrlDTO> createNewOwnInstance(
+      final ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlsToCrawl) {
+    return new HrSendungsfolgedetailsTask(crawler, aUrlsToCrawl);
   }
 
-  private Optional<LocalDateTime> parseDate(final Optional<String> aDateTimeText) {
-    if (aDateTimeText.isPresent()) {
+  @Override
+  protected void processDocument(final CrawlerUrlDTO aUrlDto, final Document aDocument) {
+    Optional<String>[] topicAndTitle = getTopicAndTitle(aDocument);
+    final Optional<String> beschreibung =
+        HtmlDocumentUtils.getElementString(DESCRIPTION_SELECTOR1, DESCRIPTION_SELECTOR2, aDocument);
+    final Map<Resolution, String> videoUrls = parseVideo(aDocument);
+    final Optional<String> untertitelUrlText =
+        HtmlDocumentUtils.getElementAttributeString(UT_URL_SELECTOR1, UT_URL_SELECTOR2, ATTRIBUTE_SRC, aDocument);
+    final Optional<Duration> dauer = parseDauer(aDocument);
+
+    if (checkPreConditions(topicAndTitle, videoUrls, dauer, aUrlDto.getUrl())) {
+
+      final Optional<LocalDateTime> time = parseDate(aDocument);
+
+      createFilm(aUrlDto, topicAndTitle[0].get(), topicAndTitle[1].get(), beschreibung, videoUrls, untertitelUrlText, dauer.get(), time);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "dauer.isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)